src/java.base/share/classes/java/util/Objects.java

```diff
@@ -486,4 +486,16 @@ long checkFromToIndex(long fromIndex, long toIndex, long length) {
     long checkFromIndexSize(long fromIndex, long size, long length) {
         return Preconditions.checkFromIndexSize(fromIndex, size, length, null);
     }
+
+    /**
+     * {@return a new instance of an unspecified class}
+     * The object has a unique identity; no other references to it exist.
```

Comment:
Is this a new javadoc style/tag ?

Reply:
Yes, it optimizes the case where the first sentence and the @ return tags have the same string.
See [JDK-8075778](https://bugs.openjdk.java.net/browse/JDK-8075778): Add javadoc tag to avoid duplication of return information in simple situations.